### ISSUE TITLE
bcmwl-kernel-source not installed correctly from AppCenter (WiFi on MacBooks)

### ISSUE THREAD

**Issue by supaeasy** [elementary/os#526]
It took me some time to check this out, maybe it is obvious but:
When bcmwl-kernel-source is available in AppCenter (MacBooks, maybe other laptops as well) it will not install successfully (altough it will say so). You will have to first install Linux Kernel Headers via Terminal and then you can use AppCenter to install these drivers. 

Steps to take:

```
sudo apt update
sudo apt-get install linux-headers-$(uname -r)
```
After this, visit AppCenter and install drivers from "Installed" Tab. WiFi will work now.

This is the solution already, I just leave this open so you might consider installing kernel headers via AppCenter or automatically before for a better user experience. Having a clean installation with no WiFi is annoying.

**Comment by aharish** [comment]
Yes, I can second that this happens and that this is the right solution to take. Although I must say that I also installed the Wi-Fi drivers in the same `apt update` command to make things slightly faster.
